FreeRTOS/FreeRTOS-Kernel
# Introduced Github Status Badge for Unit Tests

Introduced Github Status Badge for Unit Tests

Description
-----------
Just a nice little Github Status Badge  showing Unit Test Status. The badge shows status for "Main" Branch only.

Test Steps
-----------
I can see the badge is pointing to the CMock Github Actions Run and the latest run is at the top. 

Checklist:
----------


- [ x] I have tested my changes. No regression in existing tests.

Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
